topics/latency.md

```diff
@@ -75,7 +75,8 @@ or setup.
 
 We call this kind of latency **intrinsic latency**, and `valkey-cli`
 is able to measure it. This is an example run
-under Linux 3.11.0 running on an entry level server.
+under Linux 3.11.0 running on an entry level server around 2014.
+(This is old, but it illustrates how to measure this.)
```

Comment:
Yeah I tried on the laptop but I got pretty bad numbers. It needs a real machine. I opened #156.

Another option is to just not mention which hardware and kernel it was running on. The numbers could still make sense today, depending on hardware, OS and virtualized environment. A machine with lots of other loads can very well have these numbers today. What it does is actually "it will just try to measure the largest time the kernel does not provide CPU time to run to the valkey-cli process itself" and its on the microseconds scale. (I'd expect the difference to depend more on the scheduler, whether it's a real-time kernel, than anything else.)

The comment for the second run further down "The following is a run on a Linode 4096
instance running Valkey and Apache" we can change to something more generic like "in a virtualized environment sharing hardware with other loads".